src/main/resources/com/google/api/codegen/py/method_sample.snip

```diff
@@ -1,74 +1,61 @@
-@extends "py/common.snip"
-
 # Generate code samples for method-level documentation.
-@snippet generateMethodSampleCode(docConfig)
-  @let prompt = ">>>", \
-       ApiName = docConfig.getApiName, \
-       initCode = {@initCode(prompt, docConfig.getInitCode)}
-    Example:
-      @join import : docConfig.getAppImports
-        {@prompt} {@import}
-      @end
-      @if docConfig.isPageStreaming
-        {@prompt} from google.gax import CallOptions, INITIAL_PAGE
-      @end
-      {@prompt} api = {@context.upperCamelToLowerUnderscore(ApiName)}.{@ApiName}()
-      @if initCode
-        {@initCode}
-        @if docConfig.isRequestGrpcStreaming
-          {@prompt} requests = [request]
-        @end
-      @end
-      @if docConfig.isResponseGrpcStreaming
-        {@prompt} for element in {@methodCallSampleCode(docConfig, {@FALSE})}:
-        {@prompt}   # process element
-        {@prompt}   pass
-      @else
-        @if docConfig.isPageStreaming
-          {@prompt}
-          {@prompt} # Iterate over all results
-          {@prompt} for element in {@methodCallSampleCode(docConfig, {@FALSE})}:
-          {@prompt}   # process element
-          {@prompt}   pass
-          {@prompt}
-          {@prompt} # Or iterate over results one page at a time
-          {@prompt} for page in {@methodCallSampleCode(docConfig, {@TRUE})}:
-          {@prompt}   for element in page:
-          {@prompt}     # process element
-          {@prompt}     pass
-        @else
-          {@prompt} {@callResultSampleCode(docConfig.getReturnType)}\
-          {@methodCallSampleCode(docConfig, {@FALSE})}
-          @if docConfig.isLongRunningOperation
-            {@prompt}
-            {@prompt} def callback(operation_future):
-            {@prompt}     # Handle result.
-            {@prompt}     result = operation_future.result()
-            {@prompt}
-            {@prompt} response.add_done_callback(callback)
-            {@prompt}
-            {@prompt} # Handle metadata.
-            {@prompt} metadata = response.metadata()
-          @end
-        @end
-      @end
+@snippet sampleCode(apiMethod)
+  Example:
+  @join line : util.getDocLines(sampleCode_(apiMethod))
+    {@""}  >>> {@line}
```

Comment:
I am used to seeing samples prepended with `>>>`. I agree it makes it harder to copy and paste, but I see it done relatively consistently, and it makes doctesting easier.

I would not raise a stink if they were absent (this is one of those issues that I do not think really matters), but I think it is fine to have it there. The doctest support makes me even slightly more supportive. Doctesting is not favored in the Python community at this point, but autogeneration is probably a pretty strong case for it.